src/phoenix/db/insertion/span_annotation.py

```diff
@@ -0,0 +1,139 @@
+from datetime import datetime
+from typing import Any, List, Mapping, NamedTuple, Tuple
+
+from sqlalchemy import Select, and_, select, tuple_
+from sqlalchemy.ext.asyncio import AsyncSession
+from typing_extensions import TypeAlias
+
+from phoenix.db import models
+from phoenix.db.helpers import dedup
+from phoenix.db.insertion.types import (
+    Insertables,
+    Postponed,
+    Precursors,
+    QueueInserter,
+    Received,
+)
+
+_Key: TypeAlias = Tuple[str, str]
```

Comment:
```suggestion
_Key: TypeAlias = Tuple[EvalName, SpanID]
```

Some more type aliases would make it easier to understand here and the others for trace and doc annotations.

Reply:
revised. thx